app/assets/stylesheets/public_pages/about_page.scss

```diff
@@ -0,0 +1,51 @@
+body {
+  background: #ffffff;
+}
+
+main {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.title {
+  text-align: center;
+}
+
+h1 {
+  font-size: 80px;
+  margin-bottom: 0;
+}
+
+.subtitle {
+  color: red;
+}
+
+.text {
```

Comment:
you can use bootstrap "text-center", otherwise bad name for class 'text', the name doesn't describe styles applied